pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -283,6 +292,16 @@ message OperationState {
   optional k8s.io.apimachinery.pkg.apis.meta.v1.Time finishedAt = 7;
 }
 
+// ProjectRole represents a role that has access to a project
+message ProjectRole {
+  optional string name = 1;
```

Comment:
I think I requested this earlier, but roles will need a description so that humans know the purpose of a role.

Reply:
fixed